src/zui/ZUIOrganizeSidebar/SidebarListItem.tsx

```diff
@@ -0,0 +1,80 @@
+/* eslint-disable react/display-name */
+import { cloneElement, forwardRef } from 'react';
+import {
+  IconProps,
+  ListItemButton,
+  ListItemIcon,
+  Typography,
+  useTheme,
+} from '@mui/material';
+
+import messageIds from '../l10n/messageIds';
+import { useMessages } from 'core/i18n';
+
+export interface SidebarListItemProps {
+  icon: JSX.Element;
+  name: 'people' | 'projects' | 'journeys' | 'areas' | 'search';
+  open: boolean;
+  selected?: boolean;
+  onClick?: () => void;
+}
+
+const SidebarListItem = forwardRef<HTMLDivElement, SidebarListItemProps>(
```

Comment:
Is the `forwardRef` necessary? Unless I'm mistaken, the `ref` is not used anywhere?

Reply:
Or maybe it's required by `Tooltip`?

Reply:
Yes it's required to put a custom component as the child of a `Tooltip`